wallet/src/main/scala/org/bitcoins/wallet/api/WalletApi.scala

```diff
@@ -447,10 +354,11 @@ trait WalletApi extends WalletLogger {
 
 }
 
-object WalletApi {
+trait NeutrinoHDWalletApi extends HDWalletApi with NeutrinoWalletApi
 
-  case class BlockMatchingResponse(
-      blockHash: DoubleSha256DigestBE,
-      blockHeight: Int)
+trait SpvHDWalletApi extends HDWalletApi with SpvWalletApi
 
-}
+trait AnyHDWalletApi
```

Comment:
Ok, i'm confused as to what the purpose of this is. Can we get a nice description for a scaladoc?